### PR TITLE
9235 accessibility issues

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9235)

This ticket is to address two issues from accessibility.

1. Fix marriage history header - This was an h4 but this ticket changes this to an h3 to make sure that header tags follow single level progression.
2. Text inside the element is not included in the accessible name - This is on /veteran-information - This is in regards to the aria-label text not including the phone number in the same format as the text of the link. Once this aria-label is changed to match exactly in then throws an error that the aria-label does not add context to the link. Removing the aria-label removes the errors. Aria-labels are only useful when they add more context to the link as to what will happen when the user clicks, if they don't add any context then they are not useful. More info [here](https://www.w3.org/WAI/WCAG21/Techniques/general/G208.html).

## Testing done
existing unit tests still pass

## Acceptance criteria
- [x] All tasks have been addressed and a PR has been requested

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/components/ArrayPageItemSpouseTitle.jsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import React from 'react';
 export function SpouseTitle({ formData }) {
   return (
     <div>
-      <h4 className="vads-u-border-color--link-default vads-u-border-bottom--2px vads-u-margin-top--0 vads-u-padding-bottom--0p5">
+      <h3 className="vads-u-border-color--link-default vads-u-border-bottom--2px vads-u-margin-top--0 vads-u-padding-bottom--0p5">
```
Comment (Mottie) [inline src/applications/disability-benefits/686c-674/components/ArrayPageItemSpouseTitle.jsx:6]: Will this need a `vads-u-font-size--h4` class to keep the font size consistent ([ref](https://design.va.gov/utilities/font-size))?

File: src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js [hunk 2]
```diff
@@ -45,10 +45,8 @@ const VeteranInformationComponent = ({
       <p>
         <strong>Note:</strong> If you need to update your personal information,
         please call Veterans Benefits Assistance at{' '}
-        <a aria-label="800. 8 2 7. 1000" href="tel:800 827 1000">
-          800-827-1000
-        </a>{' '}
-        between 8:00 a.m. and 9:00 p.m. ET Monday through Friday.
+        <a href="tel:800 827 1000">800-827-1000</a> between 8:00 a.m. and 9:00
```
Comment (Mottie) [inline src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js:48]: Use this instead:

```js
import Telephone, { CONTACTS } from '@department-of-veterans-affairs/formation-react/Telephone';

// ...
<Telephone contact={CONTACTS.VA_BENEFITS} />
```

**Review by Mottie — APPROVED** [review]

File: src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js (hunk 2 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js]: Nice! Using this now

File: src/applications/disability-benefits/686c-674/components/ArrayPageItemSpouseTitle.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/components/ArrayPageItemSpouseTitle.jsx]: YES! Good catch

File: src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js [hunk 3]
```diff
@@ -45,10 +48,8 @@ const VeteranInformationComponent = ({
       <p>
         <strong>Note:</strong> If you need to update your personal information,
         please call Veterans Benefits Assistance at{' '}
-        <a aria-label="800. 8 2 7. 1000" href="tel:800 827 1000">
-          800-827-1000
-        </a>{' '}
-        between 8:00 a.m. and 9:00 p.m. ET Monday through Friday.
+        <Telephone contact={CONTACTS.VA_BENEFITS} /> between 8:00 a.m. and 9:00
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js:51]: So cool to see this in the wild now.

**Review by micahchiang — APPROVED** [review]

**Pull request merged**